### ISSUE TITLE
Duration in the music player is not accessible

### ISSUE THREAD

**Issue by amirmahdifard** [UnigramDev/Unigram#2882]
in the music player of the unigram, that you can see all musics wich you have in your telegram and you can download or play them, the duration of music is not accessible: only title and artist is showne: unigram vertion: 9.0 (7840) X64 direct: windows vertion: enterprise, Windows 10 21H1 (x64) build 19043.1826

**Comment by amirmahdifard** [comment]
so guys, if anyone can, please fix this bug: unigram's developer told me if i fix this bug, you gonna report another bug, but i said no, and he didn't fix this  bug, and also he muted me because he is not fixing accessibility: thank you

**Comment by Elshara** [comment]
@amirmahdifard I have tested this functionality out with music. ON its own in the chat history, everything is accessible. But if you click the open chat button, tab does not land the cursor on the relevant parts of the audio player accept for play buttons. You have to use object navigation to see what you're playing with screen readers.

**Comment by amirmahdifard** [comment]
no, this is not my  point: i'm talking about the list of that music player

**Comment by MarcAnt01** [comment]
@amirmahdifard you’re kindly invited to stop your begging even here, there are rules of conduct even here

**Comment by amirmahdifard** [comment]
what i'm doing? i provided my unigram vertion, my windows vertion, and the problem is clear, wich is you guys complane about to not fix the problems: there are 340 problem open in here, some of them from a long time ago: that means this app development is not farely: because the problems wich is for long time ago, should be fixed: and i have more stuf to say, but i won't for now:

**Comment by amirmahdifard** [comment]
what i'm doing? i provided my unigram vertion, my windows vertion, and the problem is clear, wich is you guys complane about to not fix the problems: there are 340 problem open in here, some of them from a long time ago: that means this app development is not farely: because the problems wich is for long time ago, should be fixed: and i have more stuf to say, but i won't for now:

**Comment by MarcAnt01** [comment]
Asking continuously for bug fixes is not considered acceptable, we thank you for opening issues, but you should not go any further with requests like you’ve been doing for years. Further violations may result in a direct action. End of the discussion

**Comment by amirmahdifard** [comment]
yes, i agree with you completely: but what you said is only makes sence if the developer is havving a proiority for the github problems paje, and some of them are not resulting for staying open for years: i can show you an eeshoo opened from 2018 and even before may be, so what is happening to those?

**Comment by amirmahdifard** [comment]
this issue has been fixed in Unigram 10.2.1 (9135). Thanks.
